src/core/io/src/4C_io_input_spec_builders.hpp

```diff
@@ -15,10 +15,9 @@
 #include "4C_io_input_types.hpp"
 #include "4C_io_value_parser.hpp"
 #include "4C_io_yaml.hpp"
+#include "4C_utils_enum.hpp"
 #include "4C_utils_string.hpp"
 
-#include <magic_enum/magic_enum_iostream.hpp>
```

Comment:
@dragos-ana I believe I might have fixed the problem you told me about (enums inside `InputParameterContainer` added via input mechanism not being printable). .